### ISSUE TITLE
Custom colors for nodes

### ISSUE THREAD

**Issue by WuglyakBolgoink** [swimlane/ngx-graph#55]
**I'm submitting a ...**
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

Is it possible to add in node-Object a new property `color`?

```js
{
        id: '4',
        label: 'Basic',
        color: 'red' | '#e8e8e8' | 'rgba(255, 255, 255, 0.5)'
}
```

Example what I want:
![image](https://user-images.githubusercontent.com/2271337/37199108-f1210212-2380-11e8-986e-11dd070a6451.png)

**Comment by WuglyakBolgoink** [comment]
Solution: just changed in template
```html
[attr.fill]="node.color || node.options.color"
```



```html
<ng-template #nodeTemplate let-node>
            <svg:g class="node"
                   ngx-tooltip
                   [tooltipPlacement]="'top'"
                   [tooltipType]="'tooltip'"
                   [tooltipTitle]="cc(node)">
              <svg:rect [attr.width]="node.width" [attr.height]="node.height" [attr.fill]="node.color || node.options.color"/>
              <svg:text alignment-baseline="central" [attr.x]="10" [attr.y]="node.height / 2">{{node.label}}</svg:text>
            </svg:g>
          </ng-template>
```

**Comment by NikhilCherian** [comment]
I am also working on this part. I tried to add a new color for a node as I observed in the node's json. 

data: {color: '#a27ea8'}
dimension: {width: 64.82500076293945, height: 30}
id: "tariff41"
label: "tariff_1"
meta: {forceDimensions: false}
position: {x: 241.87500190734863, y: 85}
transform: "translate(209.4625015258789, 70)"

Here, i tried to alter the color but it was not getting changed. How did you add the color property in the node? Any help is appreciated. Thanks in advance.
